cms/search/utils.py

```diff
@@ -35,8 +35,11 @@ def build_release_specific_fields(page: "Page") -> dict:
         "published": page.status == ReleaseStatus.PUBLISHED,
         "date_changes": [],
     }
-    # If page.release_date_text is present, treat it as provisional_date
+
+    # If page.release_date_text is present, treat it as a provisional_date
+    # and do not add release_date to the payload.
     if page.release_date_text:
+        data["release_date"] = None
```

Comment:
Superfluous. Why are we setting `release_date` to `None` here? This is unnecessary as we already set it to None here: https://github.com/ONSdigital/dis-wagtail/blob/5b12c7eba1ccad1251caa68b6a850706cd75aed2/cms/search/utils.py#L22

Reply:
Okay, wasn't aware, or maybe I seen some convo around this somewhere on this PR/ or somewhere else and got confused. Thanks for info, this is weird but if this is how is done, then that's fine.

Reply:
Resolving based on the comment above ^